src/components/container/container/container.scss

```diff
@@ -0,0 +1,33 @@
+@use '../../core/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host([color='white']) {
+  --sbb-container-background-color: var(--sbb-color-white-default);
+}
+
+:host([color='milk']) {
+  --sbb-container-background-color: var(--sbb-color-milk-default);
+}
+
+:host([color='midnight']) {
+  --sbb-container-background-color: var(--sbb-color-midnight-default);
+  --sbb-focus-outline-color: var(--sbb-focus-outline-color-dark);
+
+  color: var(--sbb-color-white-default);
+}
+
+.sbb-container {
+  background-color: var(--sbb-container-background-color, transparent);
+  overflow-y: auto;
```

Comment:
Probably this creates a scrollbar if a consumer sets a fixed height on the container? With this I think we remove the possibility to let the overflow be controlled by consumer from outside. I'm not really understanding why margin in a slotted content does influence the parent container (here sbb-container). hmm. Will think about it :)

Reply:
I tested it, it's ok how it is now :) sorry..